mlir/lib/Dialect/Quant/IR/TypeDetail.h

```diff
@@ -214,8 +214,9 @@ struct UniformQuantizedPerAxisTypeStorage : public QuantizedTypeStorage {
     }
   };
 
-  // We pass scales and zeroPoints in directly rather than relying on KeyTy
-  // because we have to create new reallocated versions in `construct` below.
+  // We pass quantiles, scales and zeroPoints in directly rather than relying on
```

Comment:
did you intend to change this comment to include "quantiles" inside the struct `UniformQuantizedPerAxisTypeStorage`

Reply:
good catch, wrong place, fixed now